spikeinterface/extractors/nwbextractors.py

```diff
@@ -123,10 +123,10 @@ def __init__(self, file_path: PathType, electrical_series_name: str = None, load
             unique_grp_names = list(np.unique(self._nwbfile.electrodes['group_name'][:]))
 
         # Fill channel properties dictionary from electrodes table
-        channel_ids = [self._es.electrodes.table.id[x]
-                       for x in self._es.electrodes.data]
-
-        dtype = self._es.data.dtype
+        if "channel_name" in self._nwbfile.electrodes.colnames:
+            channel_ids = self._es.electrodes["channel_name"][:].astype("str")
```

Comment:
Maybe we should ccheck that this channel_names is unique no ?

Reply:
If they come from nwb-conversion-tools they have to be unique.
Moreover, should't the initialization of BaseRecording test for uniqueness of channel_ids in line 130 below?